marshal_test.go

```diff
@@ -0,0 +1,37 @@
+package sentry
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestMarshalJSON(t *testing.T) {
+	tests := []struct {
+		in  interface{}
+		out string
+	}{
+		// TODO: eliminate empty struct fields from serialization of empty event.
```

Comment:
Today we send `{}` for empty structs that appear anywhere within an `Event`, and we can extend the `MarshalJSON` implementations to handle those cases later.